CorsixTH/Lua/persistance.lua

```diff
@@ -260,10 +260,14 @@ function SaveGameFile(filename)
   f:close()
 end
 
+--! Puts loaded file into the game
+--!param data The file
 function LoadGame(data)
   --local status, res = xpcall(function()
   local objtable = MakePermanentObjectsTable(true)
   local state = assert(persist.load(data, objtable))
+  -- Check the game we're loading is same or older than the current version
```

Comment:
No, you're checking compatibility, which today is comparing versions, and tomorrow can be different.